src/epcpm/pm_helper.py

```diff
@@ -113,20 +113,29 @@ def add_padding_to_block(
 
 def build_uuid_scale_factor_dict(
     points: typing.List[
-        typing.Union[epcpm.sunspecmodel.DataPoint, epcpm.sunspecmodel.DataPointBitfield]
+        typing.Union[
+            epcpm.staticmodbusmodel.FunctionData,
+            epcpm.staticmodbusmodel.FunctionDataBitfield,
+        ]
```

Comment:
The `build_uuid_scale_factor_dict` method still needs to work as it did before. Almost like you will have to merge the before and after.